target/linux/ath79/dts/ar7100_mikrotik_routerboard-rb4xx.dtsi

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: GPL-1.0-or-later OR MIT
+
+// Based on ar71xx arch/mips/ath79/mach-rb4xx.c
+
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar7100.dtsi"
+
+/ {
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	aliases {
+		led-boot = &led_user_yellow;
+		led-failsafe = &led_user_yellow;
+		led-running = &led_user_yellow;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_user_yellow: user_yellow {
+			label = "rb4xx:yellow:user";
+			gpios = <&gpio 4 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+		};
+
+		led_1_green: 1_green {
```

Comment:
Do you need these labels? If the name in label is "led1", I'd also use that here.
Despite, since there are no LEDs with other colors, it should be enough to name the node just "led1".

Reply:
Very true, I'll trim